src/elements/content-preview/preview-header/__tests__/PreviewHeader.test.js

```diff
@@ -31,7 +29,7 @@ describe('elements/content-preview/preview-header/PreviewHeader', () => {
         const contentAnswersProps = { show };
         const wrapper = getWrapper({ contentAnswersProps, file });
 
-        expect(wrapper.exists(ContentAnswers)).toBe(expected);
+        expect(wrapper.exists('.bcpr-PreviewHeader-contentAnswers')).toBe(expected);
```

Comment:
the component is now lazy loaded, so use classname to target since the component is now labeled as `ForwardRef` tree